tools/config_updater.py

```diff
@@ -696,6 +740,91 @@ def ManageBinaries(config=None, token=None):
   print "Please restart the service for it to take effect.\n\n"
 
 
+def SuggestedConfiguration():
```

Comment:
I was a little torn on that myself.  I went with code for a few reasons and I tried to limit this to the most significant.  I figured the documentation could be generated off the help messages to provide a comprehensive reference rather than just "Please be aware of these specific things before you hit the go button".  

1.  In the code it can display the current values.

2.  This ensures people actually at least see what their system is going to be doing before they start the system if they skip reading the docs.

3. Some of these configurations should/could become configuration prompts rather than just displays.